flow/layers/shader_mask_layer.cc

```diff
@@ -7,12 +7,12 @@
 
 namespace flutter {
 
-ShaderMaskLayer::ShaderMaskLayer(sk_sp<SkShader> shader,
+ShaderMaskLayer::ShaderMaskLayer(std::shared_ptr<DlColorSource> shader,
```

Comment:
And change blend_mode param to DlBlendMode while we're de-skiafying here...

Reply:
Done